pytradfri/api/aiocoap_api.py

```diff
@@ -177,33 +198,22 @@ async def _execute(self, api_command, timeout):
 
         return api_command.result
 
-    def debug_comm(self, call_type, api_commands):
```

Comment:
This was only used to figure out the various types of returns we could get from the gateway I presume?

Reply:
Oh, I see now that you refactored it..